cpp/velox/CMakeLists.txt

```diff
@@ -275,12 +275,9 @@ target_include_directories(velox PUBLIC
     ${JNI_INCLUDE_DIRS}
     ${CMAKE_CURRENT_SOURCE_DIR}
     ${root_directory}/src
```

Comment:
Looks like this include path has been deprecated for a long time. Could you help to remove it as well?

Reply:
ok, I wil refine it.